### ISSUE TITLE
Refactor of compute.rb fails due to instance_flavor registry not complete

### ISSUE THREAD

**Issue by rberger** [sparkleformation/sparkle-guides#5]
After doing the refactor of the compute.rb and the following is run:

```
fn update sparkle-guide-compute --file compute --defaults
```

You get:

```
Sfn]: SparkleFormation: update
[Sfn]:   -> Name: sparkle-guide-compute Path: /Users/rberger/work/omnypay/ops/sparkle/sparkle-guide/sparkleformation/compute.rb
[Sfn]: Stack runtime parameters:
[ERROR]: Not an allowed value: m1.small, m1.medium
[ERROR]: Not an allowed value: m1.small, m1.medium
[ERROR]: Not an allowed value: m1.small, m1.medium
[ERROR]: Not an allowed value: m1.small, m1.medium
[ERROR]: Not an allowed value: m1.small, m1.medium
[ERROR]: Not an allowed value: m1.small, m1.medium
[FATAL]: Failed to receive allowed parameter!
```

If  the `:instance_flavor registry entry is update to have t2.micro the above command works

```
SfnRegistry.register(:instance_flavor) do
  ['t2.micro', 't2.small', 'm1.small', 'm1.medium']
end
```

**Comment by rberger** [comment]
It looks like no matter what I do ether have:

```
SfnRegistry.register(:instance_flavor) do
  ['t2.micro', 't2.small', 'm1.small', 'm1.medium']
end
```

or

```
SfnRegistry.register(:instance_flavor) do
  ['m1.small', 'm1.medium']
end
```

When I then run 

```
sfn update sparkle-guide-compute
```

I get:

```
[Sfn]: SparkleFormation: update
[Sfn]:   -> Name: sparkle-guide-compute (no template update)
[Sfn]: Stack runtime parameters:
[Sfn]: Sparkle Image Id [ami-63ac5803]:
[Sfn]: Sparkle Ssh Key Name [dev-000]:
[Sfn]: Sparkle Flavor [t2.micro]: m1.small
[ERROR]: Not an allowed value: t2.micro, t2.small
[Sfn]: Sparkle Flavor [t2.micro]:
```

**Comment by rberger** [comment]
Looks like the issue for the second problem is the in the doc section

**The real update (parameters)**

it shows the command as:

```
sfn update sparkle-guide-compute
```

What actually works is:

```
sfn update sparkle-guide-compute --file compute
```

**Comment by rberger** [comment]
Can't go from a t2.\* to an m1.\* 

t2 are HVM and m1 aren't

**Comment by chrisroberts** [comment]
Fixed in #8. Thanks @rberger !
